sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/metadata/datasource/dialect/SQLServerDataSourceMetaData.java

```diff
@@ -32,24 +35,40 @@
  */
 @Getter
 public final class SQLServerDataSourceMetaData implements DataSourceMetaData {
-    
```

Comment:
Wrong code style. Four blank spacess are necessary at the beginning of each line, sir.